admin/app/components/user-detail-personal-information.hbs

```diff
@@ -104,8 +104,15 @@
         {{#if canAdministratorModifyUserDetails}}
         <button class="btn btn-outline-default" aria-label="Modifier" {{on "click" this.changeEditionMode}}>Modifier
         </button>
+        <button class="btn btn-outline-default btn-outline-default--danger" aria-label="Anonymiser" {{on "click" this.toggleDisplayConfirm}}>Anonymiser cet utilisateur
+        </button>
         {{/if}}
       </div>
     </form>
   {{/if}}
 </section>
+
+<ConfirmPopup @message="Etes-vous sûr de vouloir anonymiser cet utilisateur? Ceci n’est pas réversible"
+              @confirm={{action "anonymizeUser"}}
+              @cancel={{action "toggleDisplayConfirm"}}
+              @show={{displayConfirm}} />
```

Comment:
Il est de bon ton de préciser `this.` devant l'attribut, comme ça l'origine est clair. C'est même dans les recommandations Ember si je dis pas de bêtises

Reply:
Ah super merci !